modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/App.es.js

```diff
@@ -54,7 +55,13 @@ export default (props) => (
 		<AppContextProvider {...props}>
 			<ClayModalProvider>
 				<Router>
-					<div className="custom-object-app">
+					<div
+						className={classNames('custom-object-app', {
+							'publications-enabled': document.querySelector(
+								'.change-tracking-indicator'
+							),
+						})}
+					>
```

Comment:
@kevenleone @rebsilva Podemos adicionar essa div somente quando o publications for habilitado?

Reply:
@interaminense , estamos utilizando a mesma div que contém a **classe** `custom-object-app` para também exibir o `publications-enabled`, nesse caso que é a mesma div utilizar o classNames com essa implementação faz mais sentido

Reply:
Aqui nesse caso a div já existia, eu só coloquei a condição a mais de verificar se o publications está ativado (e se estiver, adiciona a classe)

Reply:
Massa, esse aqui ta ok msm!

Reply:
Feito!